portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,25 +31,105 @@
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> quotes;
-
-  @Override
-  public void init() {
-    quotes = new ArrayList<>(4);
-    quotes.add("Hope you are having a good one!");
-    quotes.add("How are you :)");
-    quotes.add("It is fantastic to see you!");
-    quotes.add("Thanks for vising my portfolio!");
-  }
+  private FeedbackRecord comment;
+  private static final int DEFAULT_MAX_N = 10;
```

Comment:
can you rename this constant to something more descriptive? what is this the max of? max comments>